### ISSUE TITLE
Can not use "targeted number"

### ISSUE THREAD

**Issue by taeyoun** [prabowomurti/themesh-solver#5]
In the mesh, once if you make the targeted number from the cells, it turn white and cannot be operated any more. But some of "Answer" from your code does not seem to consider that!! Thanks...I enjoyed using our code...being "resourceful"

**Comment by prabowomurti** [comment]
You are right, we should order the numbers so that the targeted number is not produced _before_ the calculation ends.

**Comment by taeyoun** [comment]
[😊]
